src/message.rs

```diff
@@ -1858,6 +1858,12 @@ pub enum Viewtype {
     /// A click on a sticker will offer to install the sticker set in some future.
```

Comment:
I'd rather add a comment here that it can be converted to Image using some heuristics